docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,358 @@
+
+```shell
+curl --location 'https://127.0.0.1/api/platform/v1/security/policies/bundles' \
+-H "Authorization: Bearer <access_token>"
+```
+```json
+{
+    "items": [
+        {
+            "compilationStatus": {
+                "message": "",
+                "status": "compiled"
+            },
+            "content": "",
+            "metadata": {
+                "appProtectWAFVersion": "4.815.0",
+                "attackSignatureVersionDateTime": "2024.02.21",
+                "created": "2024-06-13T09:09:10.809-07:00",
+                "modified": "2024-06-13T09:09:20-07:00",
+                "policyName": "Nginxbundletest",
+                "policyUID": "ec8681eb-1e25-4b71-93bd-b91f67c5ac99",
+                "threatCampaignVersionDateTime": "2024.02.25",
+                "uid": "de08b324-99d8-4155-b2eb-fe687b21034e"
+            }
+        }
+    ]
+}
+```
+
+{{< important >}}
+
+Take note of the *uid* field: `"uid": "de08b324-99d8-4155-b2eb-fe687b21034e"`
+
+It is one of two unique IDs we will use to download the bundle: it will be referenced as *bundle-UID*.
+
+{{< /important >}}
+
+## Download the security bundle
+
+Use a GET request to download the security bundle using the policy and bundle IDs:
+
+```shell
+curl -X GET "https://{NMS_FQDN}/api/platform/v1/security/policies/<policy-UID>/bundles/<bundle-UID>" -H "Authorization: Bearer <access token>" | jq -r '.content' | base64 -d > security-policy-bundle.tgz
+```
+
+This GET request uses the policy and bundle IDs from the previous examples:
+
+```shell
+curl -X GET -k 'https://127.0.0.1/api/platform/v1/security/policies/6af9f261-658b-4be1-b07a-cebd83e917a1/bundles/de08b324-99d8-4155-b2eb-fe687b21034e' \                                                                                                     
+    -H "Authorization: Basic YWRtaW46UncxQXBQS3lRRTRuQXRXOFRYa1J4ZFdVSWVTSGtU" \
+     | jq -r '.content' | base64 -d > security-policy-bundle.tgz
+```
+
+## Add volumes and volumeMounts to NGINX Ingress Controller
+
+In order to use the WAF security bundles, your NGINX Ingress Controller instance must have *volumes* and *volumeMounts*. Precise paths are used to detect when bundles are uploaded to the cluster.
+
+Here is an example of what must be added:
+
+```yaml
+volumes:
+- name: <volume_name>
+persistentVolumeClaim:
+    claimName: <claim_name>
+
+volumeMounts:
+- name: <volume_mount_name>
+    mountPath: /etc/nginx/waf/bundles
+```
+
+A full example of a deployment file with `volumes` and `volumeMounts` may look like the following:
+
+```yaml
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-ingress
+  namespace: nginx-ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-ingress
+  template:
+    metadata:
+      labels:
+        app: nginx-ingress
+        app.kubernetes.io/name: nginx-ingress
+     #annotations:
+       #prometheus.io/scrape: "true"
+       #prometheus.io/port: "9113"
+       #prometheus.io/scheme: http
```

Comment:
should we include commented out configuration if it's not related to the example

Reply:
Since it's intended to be a complete example, if the commented out configuration would be there, it should stay.